This repository covers the transfer learning papers for gene expression data in
human disease informatics. We presented a detailed analysis of all these works
in our paper *“Deep Transfer Learning for Human Disease Informatics based on
Gene Expression Data: Comprehensive Survey with Benchmark Results”*. Furthermore
we have benchmarked these studies against different models and parameters
settings, which is further explained in our paper.

 

| **Year** | **Paper Title**                                                                                                                                                             | **DOI**                                          | **GitHub**                                                                          |
|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------|-------------------------------------------------------------------------------------|
| 2022     | CaSee: A lightning transfer-learning model directly used to discriminate cancer/normal cells from scRNA-seq                                                                 | https://doi.org/10.1038/s41388-022-02478-5       | https://github.com/yuansh3354/CaSee                                                 |
| 2022     | scJoint: transfer learning for data integration of atlas-scale single-cell RNA-seq and ATAC-seq                                                                             | https://doi.org/10.1038/s41587-021-01161-6       | https://github.com/SydneyBioX/scJoint                                               |
| 2022     | Deep transfer learning of cancer drug responses by integrating bulk and single-cell RNA-seq data                                                                            | https://doi.org/10.1038/s41467-022-34277-7       | https://github.com/OSU-BMBL/scDEAL                                                  |
| 2021     | Transfer learning between preclinical models and human tumors identifies a conserved NK cell activation signature in anti-CTLA-4 responsive tumors                          | https://doi.org/10.1186/s13073-021-00944-5       | https://github.com/edavis71/projectR_ICI                                            |
| 2021     | XGSEA: CROSS-species gene set enrichment analysis via domain adaptation                                                                                                     | <https://doi.org/10.1093/bib/bbaa406>            | https://github.com/LiminLi-xjtu/XGSEA                                               |
| 2021     | Biologically relevant transfer learning improves transcription factor binding prediction                                                                                    | https://doi.org/10.1186/s13059-021-02499-5       | https://github.com/wassermanlab/TF-Binding-Transfer-Learning                        |
| 2020     | An interspecies translation model implicates integrin signaling in infliximab-resistant inflammatory bowel disease                                                          | https://doi.org/10.1126/scisignal.aay3258        |                                                                                     |
| 2020     | AITL: Adversarial Inductive Transfer Learning with input and output space adaptation for pharmacogenomics                                                                   | <https://doi.org/10.1093/bioinformatics/btaa442> | https://github.com/hosseinshn/AITL                                                  |
| 2020     | Conditional out-of-distribution generation for unpaired data using transfer VAE                                                                                             | <https://doi.org/10.1093/bioinformatics/btaa800> | https://github.com/theislab/trvae https://github.com/theislab/trvae_reproducibility |
| 2019     | PRECISE: a domain adaptation approach to transfer predictors of drug response from pre-clinical models to tumors                                                            | <https://doi.org/10.1093/bioinformatics/btz372>  | https://github.com/NKI-CCB/PRECISE                                                  |
| 2019     | BERMUDA: a novel deep transfer learning method for single-cell RNA sequencing batch correction reveals hidden high-resolution cellular subtypes                             | https://doi.org/10.1186/s13059-019-1764-6        | https://github.com/txWang/BERMUDA                                                   |
| 2019     | Decomposing Cell Identity for Transfer Learning across Cellular Measurements, Platforms, Tissues, and Species                                                               | <https://doi.org/10.1016/j.cels.2019.04.004>     | https://github.com/genesofeve/projectR                                              |
| 2019     | MultiPLIER: A Transfer Learning Framework for Transcriptomics Reveals Systemic Features of Rare Disease                                                                     | https://doi.org/10.1016/j.cels.2019.04.003       | (Docker Image) https://hub.docker.com/r/jtaroni/multi-plier/                        |
| 2018     | CaSTLe – Classification of single cells by transfer learning: Harnessing the power of publicly available single cell RNA sequencing experiments to annotate new experiments |  <https://doi.org/10.1371/journal.pone.0208349>  | https://github.com/yuvallb/CaSTLe                                                   |
| 2018     | Found In Translation: a machine learning model for mouse-to-human inference                                                                                                 | https://doi.org/10.1038/s41592-018-0214-9        | https://github.com/shenorrLabTRDF/FIT.mouse2man                                     |

 
